src/renderer/services/wallet/util.ts

```diff
@@ -12,12 +10,12 @@ import { WalletBalance } from '../../types/wallet'
 import { WalletBalances } from '../clients'
 import { KeystoreState, KeystoreContent, Phrase, BalanceMonoid } from './types'
 
-export const getKeystoreContent = (state: KeystoreState): Option<KeystoreContent> => pipe(state, O.chain(identity))
+export const getKeystoreContent = (state: KeystoreState): Option<KeystoreContent> => FP.pipe(state, chain(FP.identity))
 
 export const getPhrase = (state: KeystoreState): Option<Phrase> =>
-  pipe(
+  FP.pipe(
     getKeystoreContent(state),
-    O.map(({ phrase }) => phrase)
+    map(({ phrase }) => phrase)
```

Comment:
Pls keep `O` as prefix. It will help us to distinguish different `map` from other modules / libraries. Same with `chain` etc.

Here are some naming conventions (I can share my VSCode snippets to you):
```
import * as A from 'fp-ts/Array'
import * as O from 'fp-ts/lib/Option'
import * as E from 'fp-ts/lib/Either'
... // and others
```